src/main/java/seedu/address/model/event/EventFactory.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.event;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * EventFactory generates events from model state.
+ */
+public class EventFactory {
+    /**
+     * Create events from current model state.
```

Comment:
There is a Java coding standard violation, should start with `Creates` instead of `Create`.